roles/vault_operator/tasks/main.yaml

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) 2025 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+- name: Slurp remote CRD file
+  run_once: true
+  ansible.builtin.slurp:
+    src: "{{ vault_operator_crds_download_dest }}"
+  register: crd_content
+
+- name: Install CRDs
+  run_once: true
+  changed_when: false
+  kubernetes.core.k8s:
+    state: present
+    definition: "{{ crd_content['content'] | b64decode | from_yaml_all }}"
+    apply: true
+    server_side_apply:
+      field_manager: atmosphere
+      force_conflicts: true
```

Comment:
https://github.com/vexxhost/atmosphere/blob/main/roles/kube_prometheus_stack/tasks/main.yml#L292-L303

We can then do something similar to that if he vendor it.

Reply:
@mnaser used this example but pinned crds to operator's image tag. Git history shows that crds are updated often.